packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -1011,17 +1022,14 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
           return
         }
 
-        try {
-          // Inside the polling, if the order is finished, we set the order success and set the step to ORDER_SUMMARY
-          yield call(confirmOrderPoll, A.confirmOrderPoll(confirmedOrder), CARD_ORDER_POLLING)
-        } catch (e) {
-          // Exhausted the retry attempts, so just show the order summary with the order we have
-          yield put(A.confirmOrderSuccess(confirmedOrder))
+        yield call(confirmOrderPoll, A.confirmOrderPoll(confirmedOrder), CARD_ORDER_POLLING)
```

Comment:
Since we removed the `try..catch` here, the `BS_ERROR.ORDER_VERIFICATION_TIMED_OUT` error will be shown to the user. We need to make we do not simply display the string to the user.

Maybe, we shall keep `try..catch` but allow to proceed to success only if caught error is `BS_ERROR.ORDER_VERIFICATION_TIMED_OUT`.